invenio/templates/deployments/web.yaml

```diff
@@ -27,6 +27,7 @@ spec:
             "/bin/bash",
             "-c",
             "cp -R {{ .Values.web.assets.location }}/. /opt/nginx-invenio-assets/ &&
+            {{ .Values.web.containers.run_before_app }}
```

Comment:
I am worried about the unpredictable outcomes of the missing `&&`. As it is, the user is responsible for adding them. Checking for existence then looping would be more complex, but more user friendly.

At minimum, I'd describe in the readme that the && is needed.